scripts/drag_scripts/README.md

```diff
@@ -0,0 +1,18 @@
+These scripts require pyautogui and python 3.
+
+pyautogui docs: http://pyautogui.readthedocs.io/en/latest/index.html
+
+install: https://medium.com/@tracy_blog/pyautogui-installing-on-mac-os-x-86e397428b3
+
+
+Install Xcode from AppStore
+activate virtual environment (if needed)
+pip install pyobjc-core
+pip install pyobjc-framework-Quartz
+pip install image
+pip install pyautogui
+
+To use the scripts, navigate the survey to the appropriate section (drag section 1 or 2)
+
+Open terminal and run the specific script. Then, from left to right mouse over the appropriate boxes to drag objects to and from and press enter. Once all 4 or 12 (depending on which script) are entered, the script will run.
+
```

Comment:
Things that need to be elaborated on:

* which script is for which sort (three-cat vs. nine-cat)

* I run my browser and terminal full-screen, which means I need to Alt-Tab back to firefox after confirming the last click position.  I didn't realize this, so the first 10 click sequences were fired in my terminal window.  I think it's reasonable to require that both windows be visible when using this script, but that needs to be documented.

* It would probably be good to mention the mechanism this uses to help users realize what's going on.  Something like "pyautogui registers mouse positions on the screen, and can trigger clicks in those positions.  Don't move your mouse while the script is running."